.travis.yml

```diff
@@ -5,13 +5,14 @@ language: minimal
 services: docker
 
 install:
-    - docker-compose build
+    - docker-compose -f docker-compose.yaml build
```

Comment:
Github search suggests the `docker-compose.yml` naming is more common https://github.com/search?q=docker-compose.yml